<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>606</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>606</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Progetto</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="locale">
   <locale language="Italian" country="Italy"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Home">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>576</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>569</height>
     </rect>
    </property>
    <widget class="QWidget" name="PaginaPrincipale">
     <widget class="QPushButton" name="VaiPaginaAutori">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>56</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Autori</string>
      </property>
     </widget>
     <widget class="QPushButton" name="VaiPaginaConferenze">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>56</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Conferenza</string>
      </property>
     </widget>
     <widget class="QPushButton" name="VaiPaginaRiviste">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>56</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Rivista</string>
      </property>
     </widget>
     <widget class="QPushButton" name="VaiPaginaArticoli">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>56</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Articoli</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PaginaAutori">
     <widget class="QLabel" name="ID">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>96</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Identificativo</string>
      </property>
     </widget>
     <widget class="QLabel" name="Nome">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>128</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
     <widget class="QLabel" name="Cognome">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>160</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cognome</string>
      </property>
     </widget>
     <widget class="QLabel" name="TITLE">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>32</y>
        <width>184</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>AGGIUNGI AUTORE</string>
      </property>
     </widget>
     <widget class="QLabel" name="Afferenze">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>88</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Afferenze</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AggiungiAutore">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Aggiungi</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="StampaAutori">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>256</y>
        <width>781</width>
        <height>305</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="VisualizzaAutori">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>208</y>
        <width>201</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Visualizza Autori</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NomeInput">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>123</y>
        <width>125</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="IDinput">
      <property name="geometry">
       <rect>
        <x>122</x>
        <y>89</y>
        <width>125</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="CognomeInput">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>156</y>
        <width>125</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="AfferenzaInput">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>192</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="TITLE_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>48</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CREA AFFERENZA</string>
      </property>
     </widget>
     <widget class="QComboBox" name="AfferenzeBox">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>88</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="SpiegazioneAfferenze">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>64</y>
        <width>261</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>true</italic>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="text">
       <string>L'afferenza che creerai
dev'essere unica.
Essa sar√† aggiunta
nel riquadro &quot;Afferenze&quot;
dove potrai associarla
ad un autore.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CreaAfferenza">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>224</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Crea afferenza</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AggiungiAfferenza">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>160</y>
        <width>161</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Aggiungi afferenza
all'autore</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>104</y>
        <width>161</width>
        <height>57</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <italic>true</italic>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Salvi le afferenze che hai
creato ma poi devi aggiungerle
all'autore.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SvuotaAutori">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>235</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Svuota Autori</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PaginaConferenze">
     <widget class="QLineEdit" name="NomeInput_2">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>73</y>
        <width>125</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="AcronimoInput">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>106</y>
        <width>125</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="LuogoInput">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>139</y>
        <width>125</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="CalendarioConferenza">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>56</y>
        <width>231</width>
        <height>145</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>46</red>
            <green>52</green>
            <blue>54</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>46</red>
            <green>52</green>
            <blue>54</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>46</red>
            <green>52</green>
            <blue>54</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>46</red>
            <green>52</green>
            <blue>54</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>145</red>
            <green>145</green>
            <blue>145</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>46</red>
            <green>52</green>
            <blue>54</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="styleSheet">
       <string notr="true">alternate-background-color: rgb(46, 52, 54);</string>
      </property>
      <property name="gridVisible">
       <bool>true</bool>
      </property>
      <property name="horizontalHeaderFormat">
       <enum>QCalendarWidget::NoHorizontalHeader</enum>
      </property>
      <property name="verticalHeaderFormat">
       <enum>QCalendarWidget::NoVerticalHeader</enum>
      </property>
      <property name="navigationBarVisible">
       <bool>true</bool>
      </property>
      <property name="dateEditEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="numPartecipanti">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>176</y>
        <width>45</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Nome_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>42</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
     <widget class="QLabel" name="Acronimo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>112</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Acronimo</string>
      </property>
     </widget>
     <widget class="QLabel" name="Luogo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>144</y>
        <width>42</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Luogo</string>
      </property>
     </widget>
     <widget class="QLabel" name="Partecipanti">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>184</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Partecipanti</string>
      </property>
     </widget>
     <widget class="QLabel" name="Organizzatori">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>0</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>Organizzatori</string>
      </property>
     </widget>
     <widget class="QLabel" name="TITLE_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>16</y>
        <width>241</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>AGGIUNGI CONFERENZA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AggiungiConferenza">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>208</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Aggiungi</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="StampaConferenze">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>256</y>
        <width>781</width>
        <height>305</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="VisualizzaConferenze">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>216</y>
        <width>201</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Visualizza Conferenze</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SvuotaConferenze">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Svuota Conferenze</string>
      </property>
     </widget>
     <widget class="QComboBox" name="ArticoliConferenza">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>216</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>218</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Articoli</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NomePersona">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>42</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="CreaOrganizzatore">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>72</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Crea Organizzatore</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CognomePersona">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>42</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>24</y>
        <width>31</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nome </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>24</y>
        <width>51</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cognome</string>
      </property>
     </widget>
     <widget class="QListWidget" name="ListaOrganizzatori">
      <property name="geometry">
       <rect>
        <x>515</x>
        <y>95</y>
        <width>271</width>
        <height>89</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="AggiungiOrganizzatore">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>192</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Aggiungi Organizzatore</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PaginaRiviste"/>
    <widget class="QWidget" name="PaginaArticoli"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
